### ISSUE TITLE
Build command gives ios related warnings when platform defaults to android

### ISSUE THREAD

**Issue by DuBistKomisch** [isleofcode/corber#602]
```
$ npx corber build
corber: Building app
WARNING: corber
Your corber/cordova/config.xml file has set the following flag:
        <allow-navigation href="http://192.168.1.102:4200" />
This is necessary for device live-reload to work, but is unsafe and should not be used in production.
WARNING: corber
xcode project is configured for automatic signing, but no development team has been selected; project may build, but archive will fail.
cleaning up...

Built project successfully. Stored in "dist/".

Adding cordova.js to corber/cordova/www/index.html...

Linting corber/cordova/www/index.html...

0 problems
corber project built
WARNING: corber
No signing identity has been configured for your xcode project. If you have received an "ARCHIVE FAILED" message with error code 65, you will need to run "corber open ios" and set your development team (for automatic signing), or set your debug provisioning profile (for manual signing).
```

These don't show up if I pass `--platform android` explicitly:

```
 $ npx corber build --platform android
corber: Building app
WARNING: corber
Your corber/cordova/config.xml file has set the following flag:
        <allow-navigation href="http://192.168.1.102:4200" />
This is necessary for device live-reload to work, but is unsafe and should not be used in production.
cleaning up...

Built project successfully. Stored in "dist/".

Adding cordova.js to corber/cordova/www/index.html...

Linting corber/cordova/www/index.html...

0 problems
corber project built
```

**Comment by alexblom** [comment]
I'm going to leave this open for now - but I think some of the platform patches I just merged have resolved this. Will be doing a full QA pre release tomorrow.

**Comment by alexblom** [comment]
Looks to be resolved in 1.3.10 - please let me know if your experience is otherwise!

**Comment by DuBistKomisch** [comment]
not working on 1.3.10 here:

```
 $ npx corber build --verbose
corber: Building app
WARNING: corber
xcode project is configured for automatic signing, but no development team has been selected; project may build, but archive will fail.
cleaning up...

Built project successfully. Stored in "dist/".

Adding cordova.js to corber/cordova/www/index.html...

Linting corber/cordova/www/index.html...

0 problems
corber project built
WARNING: corber
No signing identity has been configured for your xcode project. If you have received an "ARCHIVE FAILED" message with error code 65, you will need to run "corber open ios" and set your development team (for automatic signing), or set your debug provisioning profile (for manual signing).
```
